src/subdomains/monitoring/entities/monitoring-balance.entity.ts

```diff
@@ -16,9 +18,36 @@ export class MonitoringBalanceEntity extends IEntity {
   @Column({ type: 'float' })
   customerBalance: number;
 
+  @Column({ type: 'float', default: 0 })
+  assetCHF: number;

```

Comment:
What is this field? Maybe we can find a better name for it? 😉

Reply:
What about "assetPriceInCHF"?